.github/draft-release-notes-config.yml

```diff
@@ -12,11 +12,17 @@ replacers:
   - search: '##'
     replace: '###'
 
+exclude-labels:
+  - 'skip-changelog'
+
+exclude-contributors:
+  - 'opensearch-drigger-bot'
+
 # Organizing the tagged PRs into unified categories
 categories:
   - title: 'Breaking Changes'
     labels:
-      - 'Breaking Changes'
+      - 'breaking'
```

Comment:
breaking-changes?

Reply:
We already have a `breaking` label but no `breaking-changes` one, is there a difference between them that warrants having both?

Reply:
It looks like a `>breaking` label [is used in the OpenSearch repo](https://github.com/opensearch-project/OpenSearch/issues?q=label%3A%3Ebreaking). For consistency maybe we should go with this?